packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -11,6 +11,7 @@ import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {LabelMedium} from "@khanacademy/wonder-blocks-typography";
 import {DROPDOWN_ITEM_HEIGHT} from "../util/constants";
+import tokens from "../../../wonder-blocks-theming/src/tokens";
```

Comment:
**issue(blocking):** This would be better if it imports that from `@ka/wb-theming` instead as the file would be directly imported and bundled as part of the `wb-dropdown` package.

This would mean that you'd need to make changes to `package.json` and other files like the TS config to include `theming` as a dependency.